src/newUi.scss

```diff
@@ -942,6 +942,24 @@ nav {
   margin: 16px 0;
 }
 
+.alert-success-container-big {
```

Comment:
These changes are also part of the other PR

Reply:
PR merged with master. Waiting Review.🙂